components/model/src/ome/model/internal/Permissions.java

```diff
@@ -205,10 +205,8 @@ public boolean isSet(Flag flag) {
 
     /**
      * returns the order of the bit representing the given {@link Flag}. This
-     * is dependent on the internal representation of {@link Permission} and
+     * is dependent on the internal representation of {@link Permissions} and
      * should only be used when necessary.
-     *
-     * @see ome.tools.hibernate.SecurityFilter
```

Comment:
In general I'd keep all forward references to our own classes. In this case, that's likely things to `ome.security` or `ome.tools`. (Which points to these potentially wanting to be decoupled :smile:)